src/components/StoryList.component.js

```diff
@@ -0,0 +1,127 @@
+import React, { Component } from 'react'
+import { Button, FormControl } from 'react-bootstrap'
+import Sidebar from 'react-sidebar'
+import './../styles/storylist.css'
+
+class StoryList extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
```

Comment:
You can set the initial state outside the constructor too. What you have still works but maybe setting initial state outside will be more consistent with the rest of the app